src/app/clients/logging-client/logging-client.ts

```diff
@@ -8,7 +8,8 @@ export const LoggingClient = {
   }: {
     userId: string;
     logs: Array<{
-      date: Date;
+      createdAt: Date;
```

Comment:
Do we need to send the `createdAt` field to the API because it doesn't look we are asking the user to input a specific date when they are creating a log entry?
- The log schema has a timeStamp feature which will automatically capture the dateTime of the writing to the database when a log is sent over.